locale/de-de/movietrailer.intent

```diff
@@ -0,0 +1,3 @@
+(spiel|spiele|beginn|beginne|start|starte) (trailer|film preview|preview) {{query}}
+(spiel|spiele|beginn|beginne|start|starte) {{query}} (trailer|film preview|preview)
```

Comment:
are some of these unstranslated or just the same in english?

Reply:
thats ok. we say this in german too. Its "eingedeutscht" in german

Reply:
then add "eingedeutscht" and any other variations that should also trigger :)

Reply:
No i mean not the word "eingedeutscht", this words are "eingedeutscht". This means that they are so common in german daily language that they where added to the german dictionary ;)